packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -110,35 +89,18 @@ export function describeHydration(
 
 /**
  * Given the schema and initial tree data, returns a hydrated tree node.
- *
+ * @remarks
  * For minimal/concise targeted unit testing of specific simple-tree content.
  *
- * TODO: determine and document if this produces "cooked" or "marinated" nodes.
+ * This this produces "marinated" nodes, meaning hydrated nodes which may not have an inner node cached yet.
  */
 export function hydrate<const TSchema extends ImplicitFieldSchema>(
 	schema: TSchema,
-	initialTree: InsertableTreeFieldFromImplicitField<TSchema>,
+	initialTree: InsertableField<TSchema> | InsertableTreeFieldFromImplicitField<TSchema>,
```

Comment:
In generic contexts, typescript can't determine that InsertableField and InsertableTreeFieldFromImplicitField give the same type, so taking both makes this easier to call from other generic methods.